pkgs/misc/opensbi/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub
+, platform ? "generic"
+}:
+
+stdenv.mkDerivation rec {
+  pname = "opensbi";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "riscv";
+    repo = "opensbi";
+    rev = "v${version}";
+    sha256 = "sha256-OgzcH+RLU680qF3+lUiWFFbif6YtjIknJriGlRqcOGs=";
+  };
+
+  makeFlags = [
+    "CROSS_COMPILE=${stdenv.cc.targetPrefix}"
```

Comment:
the CROSS_COMPILE flag does not seem to be necessary.

Reply:
You are right. I've had it since 0.9 but can't remember why I added it.